presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -8413,6 +8413,12 @@ public void testAndInFilter()
                 "values false");
     }
 
+    @Test
+    public void testArrayAggDistinct()
```

Comment:
You probably want to introduce some `set_agg` test rather than this?

Reply:
Yeah - this started out as a better implementation for array_agg(distinct) but morphed into this new agg. Will fix it.

Reply:
Done